rustfmt.toml

```
@@ -0,0 +1,64 @@
+max_width = 100
+hard_tabs = false
+tab_spaces = 4
+newline_style = "Unix"
+indent_style = "Block"
+use_small_heuristics = "Default"
```

```suggestion
use_small_heuristics = "Max"
```

perhaps? slight preference for this https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#use_small_heuristics

---

okay that's fine!